public/styles/search.css

```
@@ -0,0 +1,47 @@
+  border-radius: 32px;
+  padding: 6px 16px;
+  border: 2px solid #d1d5db;
+  color: #ffffff;
+  background-color: transparent;
+}
+#search-input::placeholder {
+  color: #ffffff;
+}
+
+.search-box {
+  display: inline-block;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)